build/initialization.sh

```diff
@@ -58,6 +58,10 @@ ControlPort 9051
 CookieAuthentication 1
 EOF
 
+cat /embassy-os/product_key.txt | tr -d '\n' | sha256sum | head -c 32 | sed 's/$/\n/' > /etc/machine-id
+rm /etc/ssh/ssh_host_*
+dpkg-reconfigure openssh-server
```

Comment:
you don't need to do this in initialization

Reply:
If I don't then there won't be any ssh_host_* keys.